Dockerfile

```diff
@@ -1,5 +1,5 @@
 # build buck from source
-FROM ubuntu:20.04 AS buck
```

Comment:
Not necessary, see https://github.com/react-native-community/docker-android/pull/178.
Can you only keep the yml change ?

Reply:
Gotcha thx ^ sure thing, done: https://github.com/react-native-community/docker-android/pull/179/commits/7a750482b9ca6777996ba16f5673b23c0f8a6c77